components/games/tuxpuck/patches/oi.patch

```diff
@@ -0,0 +1,156 @@
+--- tuxpuck-0.8.2/intro.c.orig	Fri Nov  8 13:50:29 2002
```

Comment:
Perhaps, better split patch by problem it solves or files it touches.
like, 01-timer.patch, 02-Makefile.patch, 03-png.patch